### PR TITLE
Optimize lit_char_is_unicode_letter.

### PR DESCRIPTION
**Pull request by sand1k**
Add fast path for ASCII characters which are not letters.

### CODE REVIEW

File: jerry-core/lit/lit-char-helpers.cpp [hunk 1]
```diff
@@ -112,6 +114,10 @@ lit_char_is_unicode_letter (ecma_char_t c) /**< code unit */
   {
     return true;
   }
+  else if (c <= LIT_UTF8_1_BYTE_CODE_POINT_MAX)
+  {
+    return false;
```
Comment (ruben-ayrapetyan) [inline jerry-core/lit/lit-char-helpers.cpp]: Could you, please, add a comment like `/* Other one-byte code points are not corresponding to letters */`?

**Comment by ruben-ayrapetyan** [comment]
Looks good to me

**Comment by zherczeg** [comment]
Nice idea. LGTM

File: jerry-core/lit/lit-char-helpers.cpp (hunk 1 above)
Comment (ruben-ayrapetyan) [inline jerry-core/lit/lit-char-helpers.cpp]: The comment wasn't added.

**Pull request merged**